Stability: Improve Developer Documentation (#8517)
solo-io/gloo: devel/testing/nightly-tests.md

# Nightly Tests

> Tests are also run on a schedule via a [GitHub action](/.github/workflows/nightly-tests.yaml).

Reviewer: ```suggestion
The following are run on a schedule via a [GitHub action](/.github/workflows/nightly-tests.yaml).
```

## Kubernetes End-to-End Tests

> Our [kube-e2e-tests](kube-e2e-tests.md) are run using the earlier and latest supported k8s versions. These tests use the latest release - specified with the `RELEASED_VERSION` environment variable.

Reviewer: ```suggestion
Our [kube-e2e-tests](kube-e2e-tests.md) are run using the earlier and latest supported k8s versions. These tests use the latest release - specified with the `RELEASED_VERSION` environment variable.

This is in addition to the running of kubernetes end-to-end tests as part of CI.
```
